app/javascript/helpers/apolloClient.js

```diff
@@ -0,0 +1,35 @@
+import { ApolloClient } from 'apollo-client'
+import { createHttpLink } from 'apollo-link-http'
+import { InMemoryCache, IntrospectionFragmentMatcher } from 'apollo-cache-inmemory'
+const csrfToken = document.getElementsByName('csrf-token')[0].content
+const httpLink = createHttpLink({
+  uri: '/graphql',
+  credentials: 'include',
+  headers: {
+    'X-CSRF-Token': csrfToken
+  }
+})
+
+const fragmentMatcher = new IntrospectionFragmentMatcher({
+  introspectionQueryResultData: {
+    __schema: {
+      types: [
+        {
+          kind: "INTERFACE",
+          name: "Resource",
+          possibleTypes: [
+            { name: "FileSet" },
+            { name: "ScannedResource" },
```

Comment:
TODO: add the just-merged ScannedMap here too — I created https://github.com/pulibrary/figgy/issues/1777 for this.

Reply:
I can do this on this PR if you want. Need to bump the lux version anyway.

Reply:
Actually, never mind. I would rather fix both in a separate PR.